docs/redirects.json

```diff
@@ -0,0 +1,104 @@
+{
+  "redirects": [
+    {
+      "source": "/docs/solana-playground",
+      "destination": "/docs/quickstart/solpg"
+    },
+    {
+      "source": "/docs/hello-world",
+      "destination": "/docs/quickstart/local"
+    },
+    {
+      "source": "/docs/intro-to-solana",
+      "destination": "/docs"
```

Comment:
This is a legendary one. It also seems *mostly* valid today. Maybe we should keep it somewhere and update it?

Reply:
added a [link to core concepts section](https://github.com/coral-xyz/anchor/pull/3493/commits/498c764c4a747563f290f9e62df86c25ccaa1f74#diff-60af99fc0a629b54ae36d58a81e14a28c9027473c3ec08041d11bc8e015ea3d4R7-R9) on solana-com instead, is that an okay alternative?